packages/core/test/insured-bridge/helpers/ArbitrumHelper.js

```diff
@@ -0,0 +1,14 @@
+const { web3 } = require("hardhat");
+const { toHex, toBN } = web3.utils;
+
+function applyL1ToL2Alias(l1Address) {
```

Comment:
Can you apply a similar comment to the one I requested [here](https://github.com/UMAprotocol/protocol/pull/3391/files#r715582586)